### PR TITLE
Update brand icon usage example on website

### PR DESCRIPTION
**Pull request by JustSlone**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes a comment raised within #13922
- [x] Include a change request file using `$ yarn change`

#### Description of changes

The current brand icon implementation example doesn't contain enough information to succeed in using the brand icons. In particular, without including the office-ui-fabric-core CSS the example on the page does not work. 

This change updates the CSS based example and adds another example using SVG from the same CDN.

#### Focus areas to test

The website, the brand icon page in particular (/styles/web/office-brand-icons)

### CODE REVIEW

**Comment by JustSlone** [comment]
Re-spinning codesandbox build

File: apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+  href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css"
+/>
+
+<div class="ms-BrandIcon--icon96 ms-BrandIcon--word"></div>
+```
+
+This following code shows you how to specify a 48px product icon by brand using the office-ui-fabric-core SVG and a Fluent UI `<Image>` component (a regular `<img>` element can be used too):
+
+```jsx
+import { Image } from '@fluentui/react';
```
Comment (xugao) [inline apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md:18]: what's the benefit of using `Image` here? `img` should be sufficient. I feel samples should be as simple as possible.

File: apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md [hunk 2]
```diff
@@ -1,8 +1,26 @@
 To use the Microsoft 365 multicolor brand icons, select the format and size that best meets your needs. Fluent UI includes a media query that automatically selects the right image file for the pixel density of the screen you’re targeting.
 
-The following code shows you how to specify a product icon by extension, item type, icon size, and image type using Fluent UI's `<Icon>` component:
+The following code shows you how to specify a 96px product icon by brand using the office-ui-fabric-core css and a `<div>` element:
```
Comment (xugao) [inline apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md]:
```suggestion
The following code shows you how to specify a 96px product icon by brand using the [office-ui-fabric-core](https://github.com/OfficeDev/office-ui-fabric-core) css and a `<div>` element:
```

File: apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md (hunk 1 above)
Comment (JustSlone) [reply apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md:18]: When I first tried with `` it was rendering as the size of the whole page, and it seemed like I would need some extra CSS to get it to render properly, and our brand icon example internally uses Image so I went that direction. 

However, I just tried again per your comment and this works just fine, I'll update the example. 
```jsx
<img 
  src="https://static2.sharepointonline.com/files/fabric/assets/brand-icons/product/svg/word_48x1.svg" 
  width="48"
  height="48"
  alt="Word product icon" 
/>
```

Comment (JustSlone) [reply apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md]: Fixed, thanks!

File: apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md (hunk 2 above)
Comment (JustSlone) [reply apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md]: Fixed, thanks!

**Pull request merged**